doc/changes/package_diffs/6.0.0/README.md

```diff
@@ -1,14 +1,10 @@
 # Package Version Comparison between 6.0.0 and 5.0.0
 
-Python-3.6-minimal-exasol-6.2.0 flavor was removed. Also, Standard-exasol-7.x were updated to Python3.7; with that, they need to fetch Python packages from Pypi, not Ubuntu.
-
 - [Comparison of flavor Python-3.7-minimal-exasol-6.2.0](python-3.7-minimal-EXASOL-6.2.0/README.md)
-- [Comparison of flavor Standard-exasol-7.1.0](standard-EXASOL-7.1.0/README.md)
 - [Comparison of flavor Python-3.8-minimal-exasol-6.2.0](python-3.8-minimal-EXASOL-6.2.0/README.md)
+- [Comparison of flavor R-4-minimal-exasol-6.2.0](r-4-minimal-EXASOL-6.2.0/README.md)
 - [Comparison of flavor Standard-exasol-7.0.0](standard-EXASOL-7.0.0/README.md)
+- [Comparison of flavor Standard-exasol-8.0.0](standard-EXASOL-8.0.0/README.md)
 - [Comparison of flavor Template-exasol-all-python-3.8-conda](template-Exasol-all-python-3.8-conda/README.md)
 - [Comparison of flavor Template-exasol-all-python-3.8-cuda-conda](template-Exasol-all-python-3.8-cuda-conda/README.md)
-- [Comparison of flavor Standard-exasol-8.0.0](standard-EXASOL-8.0.0/README.md)
-- [Comparison of flavor Standard-exasol-7.1.0-without-python2.7](standard-EXASOL-7.1.0-without-python2.7/README.md)
-- [Comparison of flavor R-4-minimal-exasol-6.2.0](r-4-minimal-EXASOL-6.2.0/README.md)
-
+- [Comparison of flavor Standard-exasol-7.1.0](standard-EXASOL-7.1.0/README.md)
```

Comment:
I would keep the comment here in this README that Python-3.6-minimal flavor was removed

Reply:
As discussed on the phone, it's enough to keep this info in the changelog.